manim/utils/tex_file_writing.py

```diff
@@ -251,6 +270,23 @@ def convert_to_svg(dvi_file: Path, extension: str, page: int = 1):
     return result
 
 
+def delete_nonsvg_files(additional_endings: tuple[str] = ()) -> None:
+    """Deletes the .dvi, .aux, and .log files produced when using `Tex` or `MathTex`
```

Comment:
There are more file formats that should be cleaned: when rendering with xelatex, the output is an `.xdv` file, for example.

Reply:
Is there by any chance a list of which (unneeded) files manim produces?

Reply:
My regular LaTeX editor WinEDT has a couple of more file endings which should not be relevant for the LaTeX used in Manim: 
.synctex.*
.log
.aux
.bbl
.blg
.idx
.ind
.ilg
.glo
.gls
.glg
.lof
.lot
.toc
.out

Reply:
I have just changed it so that instead of checking if a file ending is in a list (making the list effectively a blacklist), the code will check if the file ending is NOT in a list.
That way, if you want to preserve some file, it can simply be added to the whitelist instead of listing out every file ending possible.